.clang-format

```diff
@@ -6,7 +6,9 @@ AlignConsecutiveAssignments: false
 AlignConsecutiveDeclarations: false
 AlignEscapedNewlines: Left
 AlignOperands:   true
-AlignTrailingComments: true
+AlignTrailingComments:
```

Comment:
Any reason why this is changed in this PR?

Reply:
It's not intentionally, locally I have another version of clang format

Reply:
Reverted